### DISCUSSION TITLE
Will builds continue to be released?

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#159] [Q&A] [answered]
Nassif, small question, will you continue to upgrade the maintenance release from time to time, or is the idea to download it only through git?

**Comment by lfcnassif** [comment, marked as answer]
Builds will continue to be released at https://github.com/sepinf-inc/IPED/releases, assets option, non developers don't need to build from source.

Probably we will not release a 3.17.3 maintenance version, because 3.18 will be released very soon.
